_workinggroups/training.md

```diff
@@ -140,7 +140,7 @@ We are always looking for volunteers from the community to help us with our trai
 {% capture list_of_upcoming_schools %}{% include list_of_upcoming_schools.md %}{% endcapture %}
 {% capture test %}{{ list_of_upcoming_schools | strip }}{% endcapture %}
 {% if test  != "" %}
-Upcoming schools:
+<div style="margin-top: 2ex; margin-bottom: 1ex">Upcoming schools:</div>
```

Comment:
Not sure. The `big-link-container` class doesn't insert space at the end automatically because that's not always desired. But not sure why the blank lines don't do anything...